package.json

```diff
@@ -1,9 +1,15 @@
 {
   "name": "kizuna-platform",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "Friendship Game.  For Vote, Rank,  QA ...",
   "main": "index.js",
   "scripts": {
+    "build": "npm run clean && webpack --config ./webpack.prod.config.js",
+    "clean": "rm -rf build",
+    "eslint": "eslint src test",
+    "lint": "npm run eslint && npm run stylelint",
+    "start": "webpack-dev-server --devtool eval --progress --colors --hot --history-api-fallback --content-base dist",
```

Comment:
> There is special management code for the dev-server at webpack/hot/dev-server, which is automatically added by --inline. (You don’t have to add it to your webpack.config.js)

According to [hot module replacement with webpack](https://webpack.github.io/docs/hot-module-replacement-with-webpack.html), `` --inline `` is needed for HMR, shouldn't we add it?